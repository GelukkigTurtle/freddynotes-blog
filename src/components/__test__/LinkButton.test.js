import React from 'react'
import renderer from 'react-test-renderer'

import { LinkButton } from '../ButtonComponent'

describe('Header', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <LinkButton to="https://gatsby-sseon-starter.netlify.com/">
          Button
        </LinkButton>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
