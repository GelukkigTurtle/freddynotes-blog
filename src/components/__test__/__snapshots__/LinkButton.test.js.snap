// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<a
  className="link "
  href="/https:/gatsby-sseon-starter.netlify.com/"
  onClick={[Function]}
  onMouseEnter={[Function]}
>
  Button
</a>
`;
